src/compiler/crystal/tools/formatter.cr

```diff
@@ -62,7 +62,7 @@ module Crystal
     @inside_lib : Int32
     @inside_struct_or_union : Int32
     @dot_column : Int32?
-    @def_indent : Int32
+    @implicit_exception_handler_indent : Int32
```

Comment:
Would just `exception_handler_indent` be an equally good name?

Reply:
Yes, `@implicit_exception_handler_indent` looks too long to me, but `implicit` is important as same as that.